code/datums/ammo/bullet/smg.dm

```diff
@@ -145,3 +145,23 @@
 	penetration = ARMOR_PENETRATION_TIER_4
 	damage_falloff = DAMAGE_FALLOFF_TIER_6
 	scatter = SCATTER_AMOUNT_TIER_6
+
+/datum/ammo/bullet/smg/p90
+	name = "submachinegun bullet"
+
+	damage = 22
+	accurate_range = 5
+	effective_range_max = 8
+	penetration = ARMOR_PENETRATION_TIER_4
+	damage_falloff = DAMAGE_FALLOFF_TIER_6
+	scatter = SCATTER_AMOUNT_TIER_6
+
+/datum/ammo/bullet/smg/p90_twe_ap
```

Comment:
Any reason why this can't just be a variant of `/datum/ammo/bullet/smg/p90` e.g. `/datum/ammo/bullet/smg/p90/twe_ap` so you can re-use everything a p90 has and only change whats different?

Reply:
That makes sense. I'll change it.